config/nais.yml

```diff
@@ -45,7 +45,7 @@ spec:
   observability:
     logging:
       destinations:
-        - id: loki
+        - id: secure_logs
```

Comment:
er det ikke lurt å ha begge?

Reply:
Iflg https://docs.nais.io/observability/logging/how-to/team-logs/ -> 
"It is possible to use both Secure Logs and Team Logs simultaneously, but it is only recommended to do so temporarily during the migration process. Once you have confirmed that your logs are being sent correctly to Team Logs, you should remove the Secure Logs configuration from your application."

jeg tolket det som at det ikke var anbefalt annet enn i en kort overgang - så tenkte at vi kunne enable begge igjen når vi faktisk starter på migreringen.  Men kanskje det ikke er så farlig.

Reply:
team logs erstatter vel secure logs

Jeg tenker mer på at det er lurt å beholde loki for åpne logger?

Reply:
Aha, ok - jeg trodde loki og teamlogs var det samme..! Men ok, legger inn begge her da